server/tests/question.spec.js

```diff
@@ -132,16 +116,27 @@ describe('User can upvote or downvote a specific question', () => {
 	it('should return status 400 when User downvotes a question that doesn\'t exist', (done) => {
 		chai
 			.request(app)
-			.patch('/api/v1/questions/999999999999/downvote')
+			.patch('/api/v1/questions/222222222/downvote')
 			.end((err, res) => {
-				console.log(res.status);
 				expect(res).to.have.status(404);
 				expect(res.body.status).to.be.a('number').and.to.equal(404);
 				expect(res.body).to.have.property('error').and.to.be.an('string');
 				done();
 			});
 	});
 
+	it('should return status 500 when User downvotes a question that is impossible to exist', (done) => {
```

Comment:
Return 404 if the meetup does not exist. 500 is for internal server errors

Reply:
Thanks for the feedback.
I am already addressing the 404 error, but in a special case whereby the id the user enters exceeds the maximum length that is required of an integer in postgres DB.
That would cause an internal server error. 
Can using 404 still be effective in communicating the error?